nengo/params.py

```diff
@@ -304,9 +304,7 @@ def __init__(self, name, default=Unconfigurable, length=None, low=0,
         self.low = low
 
     def __set__(self, instance, value):
-        try:
-            value = tuple(int(v) for v in value)
-        except TypeError:
+        if not all(is_integer(v) for v in value):
```

Comment:
You wouldn't have that convenience in the future anyways: https://github.com/nengo/nengo/pull/1045#discussion_r71457562

I would be fine with doing the cast ourselves, but I am not sure how you can reliably detect whether a floating point number is an integer (considering rounding errors).